src/script/treeView/lib/aimaratree.ts

```diff
@@ -1,5 +1,3 @@
-// This code is based on aimaraJS - Pure Javascript TreeView Component (https://github.com/rafaelthca/aimaraJS).
```

Comment:
This comment should not be removed. Please undo this.

Reply:
Added back